codingstandards/JS/README.md

```diff
@@ -0,0 +1,13 @@
+# GTMSportswear Javascript Coding Standards
```

Comment:
Should we say "JavaScript/TypeScript" or even just "TypeScript", since that's effectively all that we use to develop new code.